src/App/Resources/en-US/Resources.resw

```diff
@@ -492,6 +492,11 @@ On retry</value>
   <data name="OutputConst.Text" xml:space="preserve">
     <value>Console Output:</value>
   </data>
+  </data>
+  <data name="OutputStack.Text" xml:space="preserve">
```

Comment:
Do we really need this ? Wouldn't it be binding by 'AutomationProperties.LabeledBy="{Binding ElementName=OutputConst}"' the above mentioned text ?